### ISSUE TITLE
Unexpected data type in response

### ISSUE THREAD

**Issue by camertron** [google/google-api-ruby-client#494]
Hey guys. First off, thanks for a great gem and for fielding the community's issues. We appreciate it!

We currently use the Android subscription API pretty extensively, and have noticed that, although the [API docs](https://developers.google.com/android-publisher/api-ref/purchases/subscriptions#resource) specify that `startTimeMillis` and `expiryTimeMillis` should be `long`s, they are presented by this gem as strings:

```ruby
result = play_api.get_purchase_subscription(package_name, product_id, token)
result.start_time_millis  # => "1459970150864"
```

Obviously we can parse the string into a number, but I thought I would create an issue in case this is unexpected behavior (it was for me at least). Is it just that `long`s are always handed back as strings?

**Comment by sqrrrl** [comment]
Known issue and something that'll be fixed in 0.10. There are a few different ways how number are described in the APIs/JSON. One of the encodings uses strings + a format specifier to indicate it's a number (mostly used for > 32 bit nums) Those aren't being coerced in to FixNum/BugNum.

Since there's a small chance it could break clients, waiting to bundle it up with some other larger changes.

**Comment by sqrrrl** [comment]
Closing as duplicate of #439
